src/components/Dashboard/SpeechRecognition/sendAudioToWatson.js

```diff
@@ -0,0 +1,29 @@
+const sendAudioToWatson = async (blob) => {
+  const headers = {
+    Authorization: `Basic ${btoa('apikey:_3ZdYo0vTNIXxSZus4DUT7KEpWC7ntgt8atW5Y8fqECI')}`,
+    'content-type': blob.type,
+  };
+
+  let result;
+  try {
+    result = await fetch(
+      'https://gateway-lon.watsonplatform.net/speech-to-text/api/v1/recognize',
+      {
+        method: 'POST',
+        headers,
+        body: blob,
+      },
+    )
+      .then(res => res.json());
+  } catch (e) {
+    return false;
+  }
+  result = result.results.filter(res => res.final);
```

Comment:
please check this part - if there's no property results in result -> user will get error